components/database/couchdb-21/files/Makefile.openindiana

```diff
@@ -1,14 +1,14 @@
-   
+ 
 install:
-	if [ -d  $(DESTDIR)$(ETCDIR) ]; then  $(RM) -r $(DESTDIR)$(ETCDIR); fi
-	if [ -d  $(DESTDIR)$(MANDIR) ]; then  $(RM) -r $(DESTDIR)$(MANDIR); fi
-	if [ -d  $(DESTDIR)$(DOCDIR) ]; then  $(RM) -r $(DESTDIR)$(DOCDIR); fi
-	/bin/rm -rf $(DESTDIR)$(RELDIR)/var
+	rm -rf $(DESTDIR)
```

Comment:
Why are you doing this? If somehow it's called with DESTDIR=/, it would be pretty interesting

Reply:
During testing I got rm -rf /*. And I installed a new build environment afterwards.